src/peergos/server/util/Postgres.java

```diff
@@ -0,0 +1,21 @@
+package peergos.server.util;
+
+import org.postgresql.Driver;
+
+import java.sql.*;
+import java.util.*;
+
+public class Postgres {
+
+    public static Connection build(String host,
+                                   int port,
+                                   String database,
+                                   String user,
+                                   String password) throws SQLException {
+        Driver driver = new Driver();
+        Properties props = new Properties();
+        props.setProperty("user", user);
+        props.setProperty("password", password);
+        return driver.connect("jdbc:postgresql://" + host + ":" + port + "/" + database, props);
```

Comment:
Just to flag: the traffic on this connection will be in the clear (and if  we want to use make use of the native Postgres SSL connection then  we need  `&ssl=true`  in the query string.

Reply:
Yep anyone using this should know to use a VPN, but I'll add a warning.